src/domains/scrobbleAlbum/partials/Tracklist.tsx

```diff
@@ -26,6 +25,17 @@ import type { Track } from 'utils/types/track';
 
 const DateTimePicker = lazyWithPreload(() => import('components/DateTimePicker'));
 
+function formatDuration(totalSeconds: number) {
+  const datetime = addSeconds(new Date(0), totalSeconds);
```

Comment:
One-argument `Date` constructor takes milliseconds, so it will be `new Date(totalSeconds * 1000);`.
Still better, I think, as it doesn't rely on external dependencies.